### PR TITLE
Few fixes in i18n files (en and fr)

### PR DESCRIPTION
**Pull request by pacodelaluna**

### CODE REVIEW

**Comment by pacodelaluna** [comment]
Last commit will fix this: https://github.com/openfoodfoundation/openfoodnetwork/issues/1376

**Comment by myriamboure** [comment]
Ping @daniellemoorhead for next release!

**Comment by myriamboure** [comment]
Awesome @pacodelaluna thank you so much for all the great job you are doing for the community :-)

**Comment by pacodelaluna** [comment]
Will also fix this issue: https://github.com/openfoodfoundation/openfoodnetwork/issues/1377

File: config/locales/en.yml [hunk 1]
```diff
@@ -318,10 +318,11 @@ en:
...
           inventory: inventory
-          text2: "If you are using the inventory tool, you can select whether new products \
-                 added by your suppliers need to be added to your inventory before they can be \
-                 stocked. If you are not using your inventory to manage your products you should \
-                 select the 'recommended' option below:"
+          text2: >
+            If you are using the inventory tool, you can select whether new products
+            added by your suppliers need to be added to your inventory before they can be
+            stocked. If you are not using your inventory to manage your products you should
+            select the 'recommended' option below:"
```
Comment (mkllnk) [inline config/locales/en.yml]: That " shouldn't be there any more. I'll fix it.

File: config/locales/en.yml [hunk 2]
```diff
@@ -351,7 +351,7 @@ en:
           visible: Visible
           not_visible: Not visible
           permalink: Permalink (no spaces)
-          permalink_tip: "This permalink is used to create the url to your shop: {{link}} your-shop-name/shop"
+          permalink_tip: "This permalink is used to create the url to your shop: %{link} your-shop-name/shop"
```
Comment (mkllnk) [inline config/locales/en.yml]: Also noticed the space that shouldn't be in the URL.

**Comment by mkllnk** [comment]
@sstead Text to look at: https://github.com/openfoodfoundation/openfoodnetwork/pull/1481/files#diff-37c9b835927e3c363588f172de9bb6fa

**Comment by sstead** [comment]
Look good @mkllnk

**Pull request merged**